### PR TITLE
Fixes all failing tests

### PR DESCRIPTION
**Pull request by smcelhinney**
Fixes failing unit tests

### CODE REVIEW

File: test/schemas/calendar.js [hunk 1]
```diff
@@ -1,53 +1,6 @@
...
-  userIds: joi.array().allow(null),
-  contentGroupId: joi.number().allow(null),
-  createContent: joi.boolean(),
-  startedDate: joi.number(),
-  publishDate: joi.number(),
-  isRecurring: joi.boolean(),
-  userId: joi.number()
-});
-
+const schemaEvent = joi.object();
```
Comment (smcelhinney) [inline test/schemas/calendar.js:3]: I removed this validation - the response changes so much that we shouldn't be validating schemas at all. Also, as we move through newer versions of the API the schema changes significantly. This wasn't the best idea by me, so I'm backtracking.

**Review by darkmavis1980 — APPROVED** [review]

**Review by mdstaff — APPROVED** [review]
Approving this without much time invested in review.

**Comment by mdstaff** [comment]
Side note there are at most 25 contributors to this repo ... would it be helpful to add any code owners or recommended code reviewers to ensure more than 2 people are able to approve PRs going forward?

**Comment by mdstaff** [comment]
Side Side note: @smcelhinney your github profile picture makes you look like Bono 🎤 🎸 😂

**Pull request merged**